enclave/enclave-api/src/main/java/com/quorum/tessera/enclave/EnclaveFactory.java

```diff
@@ -1,13 +1,45 @@
 package com.quorum.tessera.enclave;
 
-import java.util.ServiceLoader;
+import com.quorum.tessera.ServiceLoaderUtil;
+import com.quorum.tessera.config.AppType;
+import com.quorum.tessera.config.Config;
+import com.quorum.tessera.config.ServerConfig;
+import com.quorum.tessera.config.util.EnvironmentVariableProvider;
+import com.quorum.tessera.encryption.KeyManagerImpl;
+import com.quorum.tessera.encryption.KeyPair;
+import com.quorum.tessera.encryption.PublicKey;
+import com.quorum.tessera.keypairconverter.KeyPairConverter;
+import com.quorum.tessera.nacl.NaclFacadeFactory;
+import java.util.Collection;
+import java.util.Optional;
 
-public interface EnclaveFactory<C> {
+public interface EnclaveFactory {
 
-    Enclave create(C config);
+    static Enclave createServer(Config config) {
```

Comment:
This seems be the default enclave, rather than the server. I think this should be named `createDefaultEnclave` or something similar.

Reply:
Default has the same issue. I think we should rename to something else, but at least server has symmetry with 'client' for the cases where we're remoting. Either wa can be deferred as  a minor change in another change set.